external/runtime/runtime.depproj

```diff
@@ -106,6 +119,11 @@
         <_CoreCLRSymbolFiles Include="%(_CoreCLRSymbolPackagesToDownload.UnzipDestinationDir)/runtimes/$(NugetRuntimeIdentifier)/native/*.pdb" />
         <_CoreCLRSymbolFiles Include="%(_CoreCLRSymbolPackagesToDownload.UnzipDestinationDir)/runtimes/$(NugetRuntimeIdentifier)/native/*.dbg" />
         <ReferenceCopyLocalPaths Include="@(_CoreCLRSymbolFiles)"/>
+
+        <CoreCLRILFiles Include="%(_CoreCLRSymbolPackagesToDownload.UnzipDestinationDir)/runtimes/$(NugetRuntimeIdentifier)/il/*.*">
```

Comment:
Will the runtime.RID.Microsoft.Runtime.CoreCLR symbols package will already contain the pdbs? Because I don't see that you added the Private package restore.

Reply:
Yes, the package already had the needed pdbs I just needed to add CoreLib il. For reference here it is a package with all that we need https://dotnetfeed.blob.core.windows.net/dotnet-core/assets/symbols/runtime.win-x64.Microsoft.NETCore.Runtime.CoreCLR.2.2.0-preview1-26613-04.symbols.nupkg